### PR TITLE
Feature/ticket 1/project setup

### PR DESCRIPTION
**Pull request by luiggy-mamani-jalafund**
# US 1 - Project Initialization
[Go to US - Link](https://trello.com/c/NR1GSWnF/1-us-1-project-setup)

## What did I do?
- Api initializaction
- Code Style
- Global environment configuration

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1,2 @@
+bin
+obj
```
Comment (SebasthianSalpozz) [inline .gitignore]: You missed the endline of the file

File: global.json [hunk 2]
```diff
@@ -0,0 +1,6 @@
+{
+  "sdk": {
+    "rollForward": "latestMinor",
+    "version": "8.0.108"
+  }
+}
```
Comment (SebasthianSalpozz) [inline global.json]: You also missed the endline of the file :D

**Review by SebasthianSalpozz — CHANGES_REQUESTED** [review]

File: .gitignore (hunk 1 above)
Comment (luiggy-mamani-jalafund) [reply .gitignore]: what do you mean? could u give me an example?

File: .gitignore
Comment (samuel-escalera-jalafund) [inline .gitignore:1]: The .gitignore can be improved, adding other files generated automatic or generated for the IDEs, eg:
# Ignore files generated by Visual Studio
.vscode/
*.vs
*.vscode

# Ignore directories for build outputs
bin/
obj/

# Ignore Rider/IDEA project files
.idea/
*.sln.iml

# Ignore user-specific files
*.user
*.rsuser
*.suo

# Ignore Visual Studio working files
.vscode/settings.json
.vscode/tasks.json
.vscode/launch.json

# Ignore package manager output
packages/

File: .gitignore (hunk 1 above)
Comment (luiggy-mamani-jalafund) [reply .gitignore]: Done, it isn't reflected in the GitHub changes tho

Screenshot from 2024-09-16 18-18-11

File: global.json (hunk 2 above)
Comment (luiggy-mamani-jalafund) [reply global.json]: Done, it isn't reflected in the GitHub changes tho

Screenshot from 2024-09-16 18-18-23

File: .gitignore
Comment (luiggy-mamani-jalafund) [reply .gitignore:1]: Done

**Review by samuel-escalera-jalafund — APPROVED** [review]
God job

**Review by SebasthianSalpozz — APPROVED** [review]
good job

**Pull request merged**